afew/tests/test_mailmover.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: ISC
+# Copyright (c) Johannes Löthberg <johannes@kyriasis.com>
+
+import datetime
+import email
+from email.utils import format_datetime, localtime, make_msgid
+import mailbox
+import notmuch
+import os
+import shutil
+import tempfile
+import unittest
+
+
+def get_notmuch_db(db_dir):
```

Comment:
isn't that some basic plumbing for which we also have `afew/Database.py`?

Reply:
There was some reason for not using it originally, but I think I already removed that reason, so fixed.

Reply:
:+1: